### PR TITLE
Add support for PHP

### PR DESCRIPTION
**Pull request by jvoisin**
This commit was inspired (but completely rewritten) by https://blog.jmpesp.org/2020/01/fuzzing-php-with-domato.html

The php_generated.txt file was generated via parse_types.py

### CODE REVIEW

**Comment by jvoisin** [comment]
`@googlebot I signed it!

File: php/php_generated.txt [hunk 1]
```diff
@@ -0,0 +1,1172 @@
+<?php
```
Comment (ifratric) [inline php/php_generated.txt:1]: Wrong file?

Comment (jvoisin) [reply php/php_generated.txt]: Fixed

**Comment by ifratric** [comment]
Thanks, looks cool!

A couple of notes:

1. Callbacks are currently unused (`<fuzzfunction>` always point to phpinfo). Generating callback code in a similar manner to the main function could potentially expose additional bugs, especially if callbacks get access to the same variables / can mess up stuff that the caller didn't expect.

2. Currently, the return vallues from the function / method calls are ignored. In cases where function / method calls return non-trivial types, wouldn't it be better to store these return values in variables and use them as function arguments in later calls or potentially call methods on these "generated" objects?

**Comment by jvoisin** [comment]
I wanted to keep the pull-request simple.
I added your suggestions (and the other items on my todo-list) in the README.md file.

**Comment by ifratric** [comment]
Alright, merging as is for now. Thanks!

**Pull request merged**